src/System.Runtime.Caching/tests/System.Runtime.Caching/MemoryCacheTest.cs

```diff
@@ -1083,7 +1083,7 @@ public void TestCacheSliding()
                 Assert.Null(mc.Get("expire5"));
                 Assert.Equal(1, mc.GetCount());
 
-                Thread.Sleep(SlidingExpirationThresholdMSec);
+                Thread.Sleep(SlidingExpirationThresholdMSec + 1000);
```

Comment:
Is this going to be subject to spurious failures in CI?

Reply:
This particular place wasn't in its original shape (expires in 2 seconds, wait for 3 seconds) so I don't expect this to happen again. If it does - we'll figure out how to harden it.

Reply:
@KKhurin could this be mocked out instead?

Reply:
Potentially yes, but I'd like to avoid it as we will likely lose some coverage of underlying code, plus the cost of making it mockable is hard to estimate right away - I'd rather not touch this code :)